sentry/src/routes.rs

```diff
@@ -2,44 +2,80 @@
+//! All routes are implemented under the module [channel].
+//!
+//! ### Route parameters
+//!
+//! Paths which include these parameters are validated as follow:
+//!
+//!
+//! - `:id` - [`ChannelId`]
+//! - `:addr` - a valid [`Address`] or [`ValidatorId`].
+//!
+//! ### Routes
 //!
 //! - [`GET /v5/channel/list`](crate::routes::channel::channel_list)
 //!
-//! todo
+//! Request query parameters: [`ChannelListQuery`](primitives::sentry::channel_list::ChannelListQuery)
+//!
+//! Response: [`ChannelListResponse`](primitives::sentry::channel_list::ChannelListResponse)
 //!
 //! - [`GET /v5/channel/:id/accounting`](channel::get_accounting_for_channel)
 //!
-//! todo
+//! Response: [`AccountingResponse::<CheckedState>`](primitives::sentry::AccountingResponse)
 //!
 //! - [`GET /v5/channel/:id/spender/:addr`](channel::get_spender_limits) (auth required)
 //!
-//! todo
+//! Response: [`SpenderResponse`](primitives::sentry::SpenderResponse)
 //!
 //! - [`POST /v5/channel/:id/spender/:addr`](channel::add_spender_leaf) (auth required)
 //!
-//! todo
+//! This route forces the addition of a spender [`Accounting`]
+//! (if one does not exist) to the given [`Channel`] with `spent = 0`.
+//! This will also ensure that the spender is added to the [`NewState`] as well.
+//!
+//! Response: [`SuccessResponse`]
 //!
 //! - [`GET /v5/channel/:id/spender/all`](channel::get_all_spender_limits) (auth required)
 //!
-//! todo
+//! Response: [`AllSpendersResponse`](primitives::sentry::AllSpendersResponse)
+//!
+//! - [`GET /v5/channel/:id/validator-messages`][list_validator_messages]
+//!
+//!   - [`GET /v5/channel/:id/validator-messages/:addr`][list_validator_messages] - filter by the given [`ValidatorId`]
+//!   - [`GET /v5/channel/:id/validator-messages/:addr/:validator_messages`][list_validator_messages] - filters by the given [`ValidatorId`] and a
+//!     [`Validator message types`][`MessageTypes`].
+//!      - `:validator_messages` - url encoded list of [`Validator message types`][`MessageTypes`] separated by a `+`.
 //!
-//! - [`GET /v5/channel/:id/validator-messages`](channel::validator_message::list_validator_messages)
+//!         E.g. `NewState+ApproveState` becomes `NewState%2BApproveState`
 //!
-//!   - `GET /v5/channel/:id/validator-messages/:ValidatorId` - filter by ValidatorId
-//!   - `GET /v5/channel/:id/validator-messages/:ValidatorId/NewState+ApproveState` - filters by a given [`primitives::ValidatorId`] and a
-//!     [`Validator message types`](primitives::validator::MessageTypes).
+//! Request query parameters: [ValidatorMessagesListQuery](primitives::sentry::ValidatorMessagesListQuery)
 //!
-//! Request query parameters: [channel::validator_message::ValidatorMessagesListQuery]
-//! Response: [primitives::sentry::ValidatorMessageResponse]
+//! Response: [ValidatorMessagesListResponse](primitives::sentry::ValidatorMessagesListResponse)
+//!
+//! [list_validator_messages]: channel::validator_message::list_validator_messages
```

Comment:
Maybe it would be better to rename the whole mod to `validator_messages` instead for consistency?

Reply:
The paths use single not plural.
I do hesitate on the name of Response class..